test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/EmailControllerSpec.scala

```diff
@@ -61,40 +60,32 @@ class EmailControllerSpec extends ControllerSpec with AuthSupport with SessionSu
 
   implicit lazy val messagesApi: MessagesApi = controller.messagesApi
 
-  val nino                 = NINO("AB123456C")
-  val name                 = models.Name("forename", "surname")
-  val retrievedDateOfBirth = new JodaLocalDate(2000, 4, 10)
+  val nino = NINO("AB123456C")
+
+  val name = Name("forename", "surname")
+
+  val dateOfBirth = new JodaLocalDate(2000, 4, 10)
+
+  val bpr = sample[BusinessPartnerRecord]
 
   val subscriptionDetails = sample[SubscriptionDetails]
 
   def mockUuidGenerator(uuid: UUID): CallHandler0[UUID] =
     (mockUuidGenerator.nextId: () => UUID).expects().returning(uuid)
 
   def mockEmailVerification(expectedEmail: Email, expectedId: UUID, expectedName: String)(
-    result: Either[Error, EmailVerificationResponse]
-  ) =
+    result: Either[Error, EmailVerificationResponse]) =
```

Comment:
should we use the models.Name type for expectedName?

Reply:
yeah, it makes sense to I think. I'll refactor it